webapp/src/app.tsx

```diff
@@ -1887,14 +1902,28 @@ function initLogin() {
     }
 }
 
+let hidConnectionPoller: number;
```

Comment:
Is there a better place for those?

Reply:
Where would be a better place?

Reply:
TODO we should move all HID stuff under the hidbridge -- keep app.tsx smallish

Reply:
See https://github.com/Microsoft/pxt/issues/3125